tests/compas/robots/test_configuration.py

```diff
@@ -135,6 +136,33 @@ def test_fixed_length_list():
     assert len(fll) == 3
 
 
+def test_fixed_length_list_copy():
+    fll = FixedLengthList([[1, 2, 3] for _ in range(3)])
+    fll_copy = copy.copy(fll)
+    assert fll_copy[0][0] == 1
+    fll_copy[0][0] = 0
+    assert fll_copy[0][0] == 0
+    assert fll[0][0] == 0
+
+
+def test_fixed_length_list_deepcopy():
+    fll = FixedLengthList([[1, 2, 3] for _ in range(3)])
+    fll_copy = copy.deepcopy(fll)
+    assert fll_copy[0][0] == 1
+    fll_copy[0][0] = 0
+    assert fll_copy[0][0] == 0
+    assert fll[0][0] == 1
+
+
+def test_configuration_deepcopy():
+    c = Configuration([1, 2, 3], [0, 0, 0])
+    c_copy = copy.deepcopy(c)
+    assert c_copy.joint_values[0] == 1
```

Comment:
Maybe add an assertion to make sure that the copy is not the same object.
```suggestion
    assert c_copy.joint_values[0] == 1
    assert c_copy != c
```

Reply:
I'm going with `is not` just in case one day `__eq__` is written.